src/styles/style.css

```diff
@@ -9,6 +9,7 @@ body {
 /*Top Navigation*/
 
 .navbar {
+	align-items: baseline;
```

Comment:
So this is a flex item option that aligns everything to the same horizontal plane. Is that right?

Reply:
Right. I added the progress bar to this area, so must have needed to fix alignment after that.